app/src/main/java/tech/ula/ui/AppDetailsFragment.kt

```diff
@@ -8,26 +10,49 @@ import android.view.ViewGroup
 import kotlinx.android.synthetic.main.frag_app_details.*
 import tech.ula.R
 import tech.ula.model.entities.App
+import tech.ula.utils.AppsPreferences
 import tech.ula.utils.LocalFileLocator
 
 class AppDetailsFragment : Fragment() {
 
+    private lateinit var activityContext: Activity
+
     private val app: App by lazy {
         arguments?.getParcelable("app") as App
     }
 
+    private val appsPreferences by lazy {
+        AppsPreferences(activityContext.getSharedPreferences("apps", Context.MODE_PRIVATE))
+    }
+
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         return inflater.inflate(R.layout.frag_app_details, container, false)
     }
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
 
-        val activityContext = activity!!
+        activityContext = activity!!
         val localFileLocator = LocalFileLocator(activityContext.filesDir.path, activityContext.resources)
 
         apps_icon.setImageURI(localFileLocator.findIconUri(app.name))
         apps_title.text = app.name
         apps_description.text = (localFileLocator.findAppDescription(app.name))
+
+        setupPreferredServiceTypeRadioGroup()
+    }
+
+    fun setupPreferredServiceTypeRadioGroup() {
+        val appServiceTypePreference = appsPreferences.getAppServiceTypePreference(app.name)
+        if (appServiceTypePreference == "SSH") {
```

Comment:
I think these strings should be defined as constants on the `AppPreferences` class so we can change these checks to avoid string literals.